Sources/Brave/States/AppState.swift

```diff
@@ -0,0 +1,232 @@
+//
+//  AppState.swift
+//  
+//
+//  Created by Brandon T on 2023-05-15.
+//
+
+import Foundation
+import UIKit
+import BraveCore
+import Data
+import RuntimeWarnings
+import Shared
+import Growth
+import Preferences
+import Storage
+
+#if DEBUG
+import os
+#endif
+
+/// Class that does startup initialization
+/// Everything in this class can only be execute ONCE
```

Comment:
Can we put asserts to assure one time execution only? Would it be worth in your opinion